src/home/room_screen.rs

```diff
@@ -969,8 +970,12 @@ pub struct RoomScreen {
     #[rust] room_name: String,
     /// The persistent UI-relevant states for the room that this widget is currently displaying.
     #[rust] tl_state: Option<TimelineUiState>,
-    /// 5 secs timer when scroll ends
+    /// Timer to send fully read receipt
+    /// 
+    /// 5 secs timer starts when user scrolled down and scrolled_past_read_marker is true
+    /// Last displayed event will be sent as fully read receipt
```

Comment:
pro-tip: doc comments like this get rendered into Html, so you need to format them as full sentences rather than just sentence fragments across line breaks.
```suggestion
    /// Timer used to send fully-read receipts for the current room.
    ///
    /// We start a short (5 second) timer when the user scrolls down,
    /// if `scrolled_past_read_marker` is true. Then, we send a fully-read receipt
    /// for the last displayed event in this room.
```